src/content/docs/tuhura-tech/contribute.mdx

```diff
@@ -62,6 +65,12 @@ At the start of the [number].mdx files
 type: tutorial
 title: 'My Page Title'
 ```
+To import necessary styles
```

Comment:
Would these by styles or modules/components?

Reply:
Yeah think I was trying to oversimplify things, Modules works well.